src/Indexing/Indexing.jsx

```diff
@@ -0,0 +1,530 @@
+    );
+
+    const indexFilesPopupBlocks = {
+      success: indexFilesSuccessPopupBlock,
+      error: indexFilesErrorPopupBlock,
+      running: indexFilesRunningPopupBlock,
+    };
+
+
+    const downloadManifestSuccessPopupBlock = (
+      <React.Fragment>
+        <div className='index-files-popup-big-icon'>
+          <div className='index-files-circle-border'>
+            <IconComponent iconName='checkbox' dictIcons={dictIcons} />
+          </div>
+        </div>
+        <div className='index-files-popup-text'>
+          <br />
+          <p>
+            <b>Status</b>:
+            <span className='index-files-green-label'>{ this.state.downloadManifestPopupMessage }</span>
+          </p>
+          <p className='index-files-page-last-updated'>
+            Last updated: { this.state.downloadManifestStatusLastUpdated }
+          </p>
+        </div>
+      </React.Fragment>
+    );
+
+    const downloadManifestErrorPopupBlock = (
+      <div className='index-files-popup-text'>
+        <br />
+        <div className='index-files-popup-big-icon'>
+          <IconComponent iconName='status_error' dictIcons={dictIcons} />
+        </div>
+        <p>{ this.state.downloadManifestPopupMessage }</p>
+        <p>If the problem persists, please contact your commons administrator.</p>
+      </div>
+    );
+
+    const downloadManifestRunningPopupBlock = (
+      <React.Fragment>
+        <Spinner caption='' type='spinning' />
+        <div className='index-files-popup-text'>
+          <br />
+          <p><b>Status</b>: { this.state.downloadManifestPopupMessage} </p>
+          <p className='index-files-page-last-updated'>
+            Last updated: { this.state.downloadManifestStatusLastUpdated }
+          </p>
+          <br />
+          <p>It may take several minutes to generate the file manifest.</p>
+          <p>Please do not navigate away from this page until
+          the operation is complete.</p>
+        </div>
+      </React.Fragment>
+    );
+
+    const downloadManifestPopupBlocks = {
+      success: downloadManifestSuccessPopupBlock,
+      error: downloadManifestErrorPopupBlock,
+      running: downloadManifestRunningPopupBlock,
+    };
+
+    return (
+      <div className='indexing-page'>
+        <div>
+          <div className='action-panel'>
+            <div className='action-panel-title'>
+                  Index Data Files
+            </div>
+            <div className='action-panel-body'>
+              <p>An indexing file, or file manifest, is a TSV containing information about
+                  files that exist in cloud storage.
+                  Rows of importance include the MD5 sum of the file,
+                  a link to the file, the filename, and its size.</p>
+
+              <p>Upload an indexing file below to create records in indexd for new object files.</p>
+              <br />
+              <form className='index-flow-form'>
+                <input type='file' onChange={this.onChange} />
```

Comment:
```suggestion
                <input type='file' accept='.tsv' onChange={this.onChange} />
```
so we are only expecting TSV files